### ISSUE TITLE
Django 4.1: `'DatabaseSchemaEditor'` has no attribute `'sql_create_sequence'`

### ISSUE THREAD

**Issue by tony** [tbicr/django-pg-zero-downtime-migrations#38]
**Describe the bug**

```
/.venv/lib/python3.10/site-packages/django_zero_downtime_migrations/backends/postgres/schema.py", line 153, in DatabaseSchemaEditorMixin
    sql_create_sequence = PGAccessExclusive(PostgresDatabaseSchemaEditor.sql_create_sequence, use_timeouts=False)
AttributeError: type object 'DatabaseSchemaEditor' has no attribute 'sql_create_sequence'
```

[django_zero_downtime_migrations/backends/postgres/schema.py#L153](https://github.com/tbicr/django-pg-zero-downtime-migrations/blob/0.11/django_zero_downtime_migrations/backends/postgres/schema.py#L153)

Removed in django 4.1 in https://github.com/django/django/commit/2eea361eff58dd98c409c5227064b901f41bd0d6

**To Reproduce**
1. What model did you have?

https://github.com/django/django/blob/4.1.2/django/contrib/auth/base_user.py#L49

  ```python
  class AbstractBaseUser(models.Model):
      password = models.CharField(_("password"), max_length=128)
      last_login = models.DateTimeField(_("last login"), blank=True, null=True)
  
      is_active = True
  
      REQUIRED_FIELDS = []
  ```

2. How did you change the model? Did not
3. What migration were generated? None (from what I see)
4. What SQL was executed? N/A
5. What issue did you get?

```
File "/home/runner/work/project/.venv/lib/python3.10/site-packages/django/contrib/auth/models.py", line 3, in <module>
    from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
  File "/home/runner/work/project/.venv/lib/python3.10/site-packages/django/contrib/auth/base_user.py", line 49, in <module>
    class AbstractBaseUser(models.Model):
  File "/home/runner/work/project/.venv/lib/python3.10/site-packages/django/db/models/base.py", line 141, in __new__
    new_class.add_to_class("_meta", Options(meta, app_label))
  File "/home/runner/work/project/.venv/lib/python3.10/site-packages/django/db/models/base.py", line 369, in add_to_class
    value.contribute_to_class(cls, name)
  File "/home/runner/work/project/.venv/lib/python3.10/site-packages/django/db/models/options.py", line 231, in contribute_to_class
    self.db_table, connection.ops.max_name_length()
  File "/home/runner/work/project/.venv/lib/python3.10/site-packages/django/utils/connection.py", line 15, in __getattr__
    return getattr(self._connections[self._alias], item)
  File "/home/runner/work/project/.venv/lib/python3.10/site-packages/django/utils/connection.py", line 62, in __getitem__
    conn = self.create_connection(alias)
  File "/home/runner/work/project/.venv/lib/python3.10/site-packages/django/db/utils.py", line 193, in create_connection
    backend = load_backend(db["ENGINE"])
  File "/home/runner/work/project/.venv/lib/python3.10/site-packages/django/db/utils.py", line 113, in load_backend
    return import_module("%s.base" % backend_name)
  File "/opt/hostedtoolcache/Python/3.10.1/x64/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/home/runner/work/project/.venv/lib/python3.10/site-packages/django_zero_downtime_migrations/backends/postgres/base.py", line 5, in <module>
    from .schema import DatabaseSchemaEditor
  File "/home/runner/work/project/.venv/lib/python3.10/site-packages/django_zero_downtime_migrations/backends/postgres/schema.py", line 144, in <module>
    class DatabaseSchemaEditorMixin:
  File "/home/runner/work/project/.venv/lib/python3.10/site-packages/django_zero_downtime_migrations/backends/postgres/schema.py", line 153, in DatabaseSchemaEditorMixin
    sql_create_sequence = PGAccessExclusive(PostgresDatabaseSchemaEditor.sql_create_sequence, use_timeouts=False)
AttributeError: type object 'DatabaseSchemaEditor' has no attribute 'sql_create_sequence'
Traceback (most recent call last):
  File "/home/runner/work/project/scan_migrations.py", line 120, in <module>
    exit_code = check_migrations()
  File "/home/runner/work/project/scan_migrations.py", line 96, in check_migrations
    if should_skip_migrations():
  File "/home/runner/work/project/scan_migrations.py", line 35, in should_skip_migrations
    migrations = subprocess.check_output(
  File "/opt/hostedtoolcache/Python/3.10.1/x64/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/hostedtoolcache/Python/3.10.1/x64/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command 'poetry run ./manage.py migrate --plan' returned non-zero exit status 1.
```

**Expected behavior**

Shouldn't fail when checking migrations

**Versions:**
 - Postgres: 13
 - Python: 3.11
 - Django: 3.1.12
 - django-pg-zero-downtime-migrations library: 0.11

**Comment by tony** [comment]
@tbicr I'm not sure how this impacts locking (if at all). I may need your assistance on this ☝️

**Comment by tbicr** [comment]
fixed in #42
